### DISCUSSION TITLE
SKIDL and Altium

### DISCUSSION THREAD

**Discussion by sebo83910** [xesscorp/skidl#196] [General]
I've just discovered this package.
Is there any possibility to use it with Altium Designer instead of KiCad?
Thanks.

Sebastien

**Comment by devbisme** [comment]
Hi, Sebastien. SKiDL doesn't support Altium right now.

SKiDL would need two things to support Altium: 1) a parser for the Altium libraries, and 2) a function to generate Altium netlists. I can put the scaffolding in place to support those, but somebody else would have to write the detailed code that performs these operations. I don't have access to Altium, and I don't have the time or energy to do this even if I did.
